tools/nodeset_compiler/backend_open62541_datatypes.py

```diff
@@ -40,7 +46,15 @@ def generateXmlElementCode(value, alloc=False, max_string_length=0):
     return "UA_XMLELEMENT{}({})".format("_ALLOC" if alloc else "", splitStringLiterals(value, max_string_length=max_string_length))
 
 def generateByteStringCode(value, alloc=False, max_string_length=0):
-    return "UA_BYTESTRING{}({})".format("_ALLOC" if alloc else "", splitStringLiterals(value, max_string_length=max_string_length))
+   return "char stringArr?{}! = {};\n\
+    UA_ByteString variable;\n\
+    variable.length = {};\n\
+    variable.data = (UA_Byte *)UA_malloc(variable.length);\n\
```

Comment:
Add a check if `variable.data` is NULL and return `UA_STATUSCODE_BADOUTOFMEMORY`.